apps/meteor/app/livechat/server/hooks/markRoomResponded.ts

```diff
@@ -12,7 +12,7 @@ export async function markRoomResponded(
 	room: IOmnichannelRoom,
 	roomUpdater: Updater<IOmnichannelRoom>,
 ): Promise<IOmnichannelRoom['responseBy'] | undefined> {
-	if (message.t || isEditedMessage(message) || isMessageFromVisitor(message)) {
+	if (message.t || isEditedMessage(message) || isMessageFromVisitor(message) || isSystemMessage(message)) {
```

Comment:
I don't think there is any real difference in this change, we are already testing whether the `t` exists here